src/Factory.php

```diff
@@ -338,29 +343,37 @@ protected static function isFileZipSource($source)
 
     protected static function loadHttpZipSource($source)
     {
+        print_r('calling loadHttpZipSource');
```

Comment:
Please remove this print

Reply:
Oh, so sorry. Gosh. Addressed and committed